sql/2025/webassembly/instruction_kinds.sql

```diff
@@ -0,0 +1,33 @@
+# Query for instructions kinds
+
+WITH wasm AS (
+  SELECT
+    *,
+    JSON_QUERY(payload, '$._wasm_stats') AS wasm_stats
+  FROM
+    `httparchive.crawl.requests`
+  WHERE
+    date = '2025-06-01' AND (type = 'wasm')
+)
+
+SELECT
+  client,
+  SUM(SAFE_CAST(JSON_VALUE(wasm_stats, '$.instr.categories.other') AS INT64)) / SUM(SAFE_CAST(JSON_VALUE(wasm_stats, '$.instr.total') AS INT64)) AS other_pct,
```

Comment:
FYI you can simplify all `JSON_VALUE` calls to one parameter (without quotes):

```suggestion
  SUM(SAFE_CAST(JSON_VALUE(wasm_stats.instr.categories.other) AS INT64)) / SUM(SAFE_CAST(JSON_VALUE(wasm_stats, '$.instr.total') AS INT64)) AS other_pct,
```